tests/vmi_multus_test.go

```diff
@@ -53,6 +53,8 @@ const (
 	ptpConfCRD             = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s"},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"mynet\", \"plugins\": [{\"type\": \"ptp\", \"ipam\": { \"type\": \"host-local\", \"subnet\": \"10.1.1.0/24\" }},{\"type\": \"tuning\"}]}"}}`
 	sriovConfCRD           = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s","annotations":{"k8s.v1.cni.cncf.io/resourceName":"%s"}},"spec":{"config":"{ \"name\": \"sriov\", \"type\": \"sriov\", \"ipam\": { \"type\": \"host-local\", \"subnet\": \"10.1.1.0/24\" } }"}}`
 	sriovLinkEnableConfCRD = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s","annotations":{"k8s.v1.cni.cncf.io/resourceName":"%s"}},"spec":{"config":"{ \"name\": \"sriov\", \"type\": \"sriov\", \"link_state\": \"enable\", \"ipam\": { \"type\": \"host-local\", \"subnet\": \"10.1.1.0/24\" } }"}}`
+	sriovConfVlanCRD       = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s","annotations":{"k8s.v1.cni.cncf.io/resourceName":"%s"}},"spec":{"config":"{ \"name\": \"sriov\", \"type\": \"sriov\", \"link_state\": \"enable\", \"vlan\": 200, \"ipam\": {}}"}}`
+	sriovConfNoVlanCRD     = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s","annotations":{"k8s.v1.cni.cncf.io/resourceName":"%s"}},"spec":{"config":"{ \"name\": \"sriov\", \"type\": \"sriov\", \"link_state\": \"enable\", \"vlan\": 0, \"ipam\": {}}"}}`
```

Comment:
\"vlan\": 0 is redundant here.

Reply:
I don't see `vlan` in `sriovConfCRD` and `sriovConfNoVlanCRD`. How it works?
But that's not a big deal, you can keep it.

Reply:
@AlonaKaplan this is a hack for creating NetworkAttachmentDefinition for sriov network, 
these constants variables are actually json strings of NAD objects, we use to `createNetwork` to create NAD by invoking http post request to the NAD API in the cluster.
The proper way to create sriov network is creating `SriovNetwork` object that the sriov-operator will reconcile and create corresspond NAD object, this PR #3551 imports SriovNetwork library and use it instead the current method.

For `linuxBridgeConfCRD` `ptpConfCRD` these are also NAD json string, it is necessary to extend `kubecli.KubevirtClient` so it will be possible to create `NetworkAttachmentDefinition` objects with the client directly.